test/recipes/30-test_evp_data/evpciph_aes_wrap.txt

```diff
@@ -7,12 +7,12 @@
 # https://www.openssl.org/source/license.html
 
 # AES wrap tests from RFC3394
-Cipher = id-aes128-wrap
+Cipher = aes128-wrap
```

Comment:
These were here deliberately to test the aliases..

Reply:
So, @richsalz please keep the aliases but add a comment so it is not reverted again.

Reply:
Added comment, will copy tests with different case names and comment those.  Updated commit pushed.